### PR TITLE
issue: 3899827: Updating TFS documentations for 1.0.0-14

### PR DESCRIPTION
**Pull request by ananalaghbar**

### CODE REVIEW

**Review by vg12345 — APPROVED** [review]

File: plugins/fluentd_telemetry_plugin/README.md [hunk 1]
```diff
@@ -2,142 +2,254 @@ UFM Telemetry endpoint stream To Fluentd endpoint (TFS)
...
+|    [streaming.stream_only_new_samples](conf/fluentd_telemetry_plugin.cfg#L16)    |   True   |                                                                                                    If True, the plugin will stream only the changed values [Default is True]                                                                                                     |
+|            [streaming.enabled](conf/fluentd_telemetry_plugin.cfg#L17)            |   True   |                                                                                     If True, the streaming will be started once the required configurations have been set [Default is False]                                                                                     |
+|       [logs-config.logs_file_name](conf/fluentd_telemetry_plugin.cfg#L20)        |   True   |                                                                                                                     Log file name [Default = '/log/tfs.log']                                                                                                                     |
```
Comment (TalLerner) [inline plugins/fluentd_telemetry_plugin/README.md:209]: Why is this true?

File: plugins/fluentd_telemetry_plugin/README.md [hunk 2]
```diff
@@ -2,142 +2,254 @@ UFM Telemetry endpoint stream To Fluentd endpoint (TFS)
...
+|         [logs-config.logs_level](conf/fluentd_telemetry_plugin.cfg#L22)          |   True   |                                                                                                                                Default is 'INFO'                                                                                                                                 |
```
Comment (TalLerner) [inline plugins/fluentd_telemetry_plugin/README.md:210]: same

File: plugins/fluentd_telemetry_plugin/README.md [hunk 3]
```diff
@@ -2,142 +2,254 @@ UFM Telemetry endpoint stream To Fluentd endpoint (TFS)
...
+|      [logs-config.max_log_file_size](conf/fluentd_telemetry_plugin.cfg#L24)      |   True   |                                                                                                                Maximum log file size in Bytes [Default is 10 MB]                                                                                                                 |
```
Comment (TalLerner) [inline plugins/fluentd_telemetry_plugin/README.md:211]: same

File: plugins/fluentd_telemetry_plugin/README.md [hunk 4]
```diff
@@ -2,142 +2,254 @@ UFM Telemetry endpoint stream To Fluentd endpoint (TFS)
...
+|    [logs-config.log_file_backup_count](conf/fluentd_telemetry_plugin.cfg#L26)    |   True   |                                                                                                                Maximum number of backup log files [Default is 5]                                                                                                                 |
```
Comment (TalLerner) [inline plugins/fluentd_telemetry_plugin/README.md:212]: same

File: plugins/fluentd_telemetry_plugin/README.md [hunk 5]
```diff
@@ -2,142 +2,254 @@ UFM Telemetry endpoint stream To Fluentd endpoint (TFS)
...
+
+![image](TFS_Architecture.png)
+
 
 Overview
 --------------------------------------------------------
 
 NVIDIA UFM Telemetry platform provides network validation tools to monitor network performance and conditions, capturing and streaming rich real-time network telemetry information, application workload usage to an on-premise or cloud-based database for further analysis.
-As a fabric manager, the UFM Telemetry holds a real-time network telemetry information of the network topology. This information should be reflected, over time (as it can change with time) towards telemetry console. In order to do so, we present stream the UFM Telemetry data To the [Fluentd](https://www.fluentd.org/) plugin
+As a fabric manager, the UFM Telemetry holds real-time network telemetry information of the network topology. This information should be reflected, over time (as it can change with time) towards telemetry console. In order to do so, we present stream the UFM Telemetry data To the [Fluentd](https://www.fluentd.org/) plugin
```
Comment (TalLerner) [inline plugins/fluentd_telemetry_plugin/README.md:14]: typo To=> to

File: plugins/fluentd_telemetry_plugin/README.md (hunk 1 above)
Comment (ananalaghbar) [reply plugins/fluentd_telemetry_plugin/README.md:209]: The `Required` column in this table indicates that the parameter cannot be empty.

**Review by TalLerner — APPROVED** [review]

**Pull request merged**